specification/cognitiveservices/AnomalyDetector/client.tsp

```diff
@@ -4,3 +4,14 @@ import "./main.tsp";
 using Azure.ClientGenerator.Core;
 
 @@convenientAPI(AnomalyDetector.Univariate.DetectUnivariateEntireSeries, false)
+
+@@projectedName(AnomalyDetector.Multivariate.ModelDetails.dataSource, "java", "dataSourceUri");  // Prefer a different name for java only
+@@projectedName(AnomalyDetector.Multivariate.ModelDetails.dataSource, "csharp", "dataSourceUri"); // Prefer a different name for C# only
+@@projectedName(AnomalyDetector.Multivariate.ModelDetails.endTime, "csharp", "endsOn"); // Prefer a different name for C# only
+@@projectedName(AnomalyDetector.Multivariate.ModelDetails.startTime, "csharp", "starsOn"); // Prefer a different name for C# only
+@@projectedName(AnomalyDetector.Multivariate.DetectionValue.interpretation, "csharp", "interpretations"); // Prefer a different name for C# only
```

Comment:
I assume Java need this as well?
https://apiview.dev/Assemblies/Conversation/66bfc5cd130a4eb4a4e05b9d192c7e2b#d7c8e26a429f47f7b07a3bfead864554